zmon-common/src/main/java/org/zalando/zmon/domain/ServiceLevelStatus.java

```diff
@@ -0,0 +1,113 @@
+package org.zalando.zmon.domain;
+
+import java.util.HashMap;
+
+public class ServiceLevelStatus {
+    private ServiceLevelStatusData data;
+    private String lastModified;
+    private String lastModifiedBy;
+
+    public ServiceLevelStatusData getData() {
+        if (data == null) {
+            return new ServiceLevelStatusData();
+        }
+        return data;
+    }
+
+    public void setData(ServiceLevelStatusData data) {
+        this.data = data;
+    }
+
+    public String getLastModified() {
+        return lastModified;
+    }
+
+    public void setLastModified(String lastModified) {
+        this.lastModified = lastModified;
+    }
+
+    public String getLastModifiedBy() {
+        return lastModifiedBy;
+    }
+
+    public void setLastModifiedBy(String lastModifiedBy) {
+        this.lastModifiedBy = lastModifiedBy;
+    }
+
+
+    public static class ServiceLevelStatusData {
+        private Integer ingestMaxCheckTier = 3;
+        private Integer queryDistanceHoursLimit = 0;
+        private Integer queryMaxCheckTier = 3;
+        private final HashMap<Integer, String> checkTiers;
+        private String message;
+
+        public ServiceLevelStatusData() {
+            this.checkTiers = new HashMap<>();
+            this.checkTiers.put(2, "metrics classified as \"important\" and \"critical\"");
+            this.checkTiers.put(1, "metrics classified as \"critical\"");
+        }
+
+        public String getMessage() {
+            return message;
+        }
+
+        public void setMessage() {
```

Comment:
This method seems for me a bit confusing. This class is, basically, a DTO, and they, by design, should not contain business logic (or any other logic).
Plus the name of the method looks like setter, but it is not a setter. 
Let me advise two options:
 - Just rename the method and call it before mapping object to JSON (as it is now). The disadvantage of such approach is that you could forget to make this call.
 - Create a post-construct method, which would have this logic. Some insights could be found here: https://stackoverflow.com/questions/6834677/jackson-mapper-post-construct

Reply:
Renamed the method to fillMessage